rdr_service/dao/biobank_specimen_dao.py

```diff
@@ -0,0 +1,98 @@
+                                                    ('confirmationDate', 'confirmedDate'),
+                                                    ('studyID', 'studyId'),
+                                                    ('repositoryID', 'repositoryId'),
+                                                    ('rlimsID', 'rlimsId'),
+                                                    ('testcode', 'testCode')]:
+            result[client_field_name] = result.pop(model_field_name)
+
+        # Translate biobankId
+        result['participantID'] = to_client_biobank_id(result.pop('biobankId'))
+
+        # Format dates
+        for date_field in ['collectionDate', 'confirmationDate', 'processingCompleteDate', 'created']:
+            format_json_date(result, date_field)
+
+        result_status = result['status']
+        result['status'] = {}
+        for status_field in ['deviations', 'freezeThawCount', 'location', 'processingCompleteDate', 'quantity',
+                             'quantityUnits']:
+            if status_field in result:
+                result['status'][status_field] = result.pop(status_field)
+        result['status']['status'] = result_status
+
+        return result
+
+    #pylint: disable=unused-argument
+    def from_client_json(self, resource, id_=None, expected_version=None, participant_id=None, client_id=None):
+        order = BiobankSpecimen(rlimsId=resource['rlimsID'], orderId=resource['orderID'], testCode=resource['testcode'],
+                                biobankId=from_client_biobank_id(resource['participantID']))
+
+        if not self.exists(resource):
+            order.created = clock.CLOCK.now()
+
+        for client_field, model_field, parser in [('repositoryID', 'repositoryId', None),
+                                                  ('studyID', 'studyId', None),
+                                                  ('cohortID', 'cohortId', None),
+                                                  ('sampleType', 'sampleType', None),
+                                                  ('collectionDate', 'collectionDate', parse_date),
+                                                  ('confirmationDate', 'confirmedDate', parse_date)]:
+            self.map_optional_json_field_to_object(resource, order, client_field, model_field, parser)
+
+        if 'status' in resource:
+            for status_field_name, parser in [('status', None),
+                                              ('freezeThawCount', None),
+                                              ('location', None),
+                                              ('quantity', None),
+                                              ('quantityUnits', None),
+                                              ('deviations', None),
+                                              ('processingCompleteDate', parse_date)]:
+                self.map_optional_json_field_to_object(resource['status'], order, status_field_name, parser=parser)
+
+        order.version = 1
```

Comment:
Do we have `version` for this model?

Reply:
We don't. I'll remove this.

Reply:
So I'm finding that removing this causes issues in BaseApi's _make_response method https://github.com/all-of-us/raw-data-repository/blob/devel/rdr_service/api/base_api.py#L301. That method attempts to access a version field on the object. Is it better to leave it there, or to set up a system of configuring the BaseApi to ignore version for specimen endpoints?